### PR TITLE
Example pages

### PR DESCRIPTION
**Pull request by nstjean**
This is just some cleanup. I organized and labeled the required dependencies on each of the example map pages, as well as making them the same between the three of them and in the readme.

In the readme I've moved the "Browse layers" dependencies up to the main "Dependencies" section. I've also renamed it to "New Layers Menu" for clarity.

### CODE REVIEW

**Comment by nstjean** [comment]
@crisner It looks like the cypress tests are all failing. Can you take a look?

**Comment by crisner** [comment]
Wow! All Cypress specs seem to have failed. These tests use index.html. Can you check if anything's missing or anything has been changed that is affecting the tests? 

Are the tests passing locally? If yes, then it must be a timing issue. If they are failing locally and if there are no changes affecting the changes you could try to increase the wait time to see if they are passing.

**Comment by nstjean** [comment]
Ok I got most of them fixed, it was an issue with a changed include script.

However the fracTrackerMobile minimal mode tests are failing for me even on the master branch.

leaflet-environmental-layers - Google Chrome_064

**Comment by nstjean** [comment]
Oh sure, and it goes and passes, haha! Ok this is all set! @cesswairimu

File: README.md [hunk 1]
```diff
@@ -230,12 +230,14 @@ The optional options object can be passed in with any of the following propertie
 #### Dependencies
 
 ```
-  <!-- Bootstrap --> 
-  <link rel="stylesheet" href="../node_modules\bootstrap\dist\css\bootstrap.min.css">
   <script src="../node_modules\jquery\dist\jquery.min.js"></script>
   <script src="../node_modules\bootstrap\dist\js\bootstrap.min.js"></script>
+<<<<<<< HEAD
```
Comment (crisner) [inline README.md]: Looks like this slipped through during a merge conflict fix?

Comment (nstjean) [reply README.md]: Oh yeah, it sure did. And it looks like there were some in there from before. I think I fixed it all.

File: cypress/integration/eonetFiresLayer.spec.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 describe('Eonet fires layer', function() {
   it('adds markers on click', function() {
-    cy.openWindow('/example/index.html#lat=43.00&lon=-83.00&zoom=3&layers=Standard')
+    cy.openWindow('/example/index.html#lat=43.00&lon=-80.00&zoom=4&layers=Standard')
```
Comment (crisner) [inline cypress/integration/eonetFiresLayer.spec.js]: :tada:

**Pull request merged**